diplomacy/web/src/diplomacy/utils/utils.js

```diff
@@ -197,4 +197,9 @@ export const UTILS = {
         OPPONENT_MOVE: 8,
     },
 
+    Stance: {
+        NON_ALLY: 0,
+        ALLY: 1,
+    }
```

Comment:
I meant that these should be actual strings so the stance is easier to interpret from the sent JSON:

```suggestion
    Stance: {
        NON_ALLY: "non-ally",
        ALLY: "ally",
    }
```